pkg/rules/celsig/wrapper/event.proto

```diff
@@ -0,0 +1,50 @@
+// This wrapper is a quick win for the POC to integrate trace.Event with cel-go without implementing ref.TypeProvider for Go structs.
+//
+// $ sudo apt-get install --yes protobuf-compiler
+// $ go install github.com/golang/protobuf/protoc-gen-go
+// $ export PATH=$PATH:$(go env GOPATH)/bin
+// $ protoc pkg/rules/celsig/wrapper/event.proto --go_out=plugins=grpc:/vagrant
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package wrapper;
+
+option go_package = "pkg/rules/celsig/wrapper";
+
+message Event {
```

Comment:
So we will not support header access here?

Reply:
TBD. This proto message is just a quick win and shortest path to evaluate cel-go implementation and benchmark it. How we model Events and how we use Protocol Buffer messages will probably be a separate PR.